int32 antOffsetX
int32 antOffsetY
int32 antOffsetZ
